.go-version

```diff
@@ -0,0 +1 @@
+1.15.1
```

Comment:
This is undocumented, but mentioned in https://github.com/netlify/build-image/issues/205